src/main/java/frc/robot/commands/SwerveTrajectoryFollower.java

```diff
@@ -0,0 +1,59 @@
+package frc.robot.commands;
+
+import edu.wpi.first.math.controller.ProfiledPIDController;
+import edu.wpi.first.math.trajectory.Trajectory;
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.SwerveControllerCommand;
+import frc.robot.Constants;
+import frc.robot.Constants.Swerve;
+
+public class SwerveTrajectoryFollower extends CommandBase{
```

Comment:
What is the difference `SwerveControllerFollower` and `SwerveTrajectoryFollower`, seems like a class that got somehow duplicated. I believe that ``SwerveControllerFollower` is the command you want.

Reply:
Removed bad one, fixed!